src/chunks/advertise-template-chunk/advertise-price-template-chunnk/AdvertisePriceChunk.tsx

```diff
@@ -0,0 +1,123 @@
+              >
+                <li className="gap-[6px] flex items-center ">
+                  <Image
+                    src={p.img}
+                    alt="good"
+                    className="w-[32px] h-[32px] "
+                  />{" "}
+                  <h1 className="font-[600] font-epilogue text-[#1B1B1B] text-[14px] leading-[22px]">
+                    {p.text}
+                  </h1>
+                </li>
+              </ul>
+            ))}
+
+            {/* second divider */}
+            <div className="w-[320px]  border-[1px] border-[#D6DDEB] mx-[0.7rem]"></div>
+
+            <div className="flex flex-col items-center justify-center  gap-[10px] my-[2rem]  ">
+              <h1 className="font-[700] text-[#002360] text-[30px] leading-[48px] font-epilogue">
+                #20,000
+              </h1>
+              <button
+                className="bg-[#0046BF] md:w-[147px] md:h-[42px] gap-[10px] text-[#FFFFFF] 
+md:leading-[22px] md:text-[14px] font-[700] font-epilogue md:pt-[10px] md:pr-[10px] md:pb-[10px] md:pl-[10px] "
+              >
+                Make a Payment
+              </button>
+            </div>
+          </div>
+        </div>
+
+        {/* Premium Plan */}
+        <div className="bg-div-custom w-[87%]   rounded-[20px]">
+          <div className="lg:w-[350px] rounded-[20px] border-[1px] bg-[#FFFFFF] border-[#D6DDEB] mx-[1rem] my-[1rem]">
+            <div className="flex flex-col gap-[10px] p-[1.5rem] mx-[0.3rem]">
+              <h1 className="font-clash font-[600] text-[24px] leading-[40px]  text-[#002360]">
+                Premium Plan
+              </h1>
+              <p className="font-[500] text-[16px] leading-[32px] font-epilogue text-[#1B1B1B99]">
+                Ideal for large businesses{" "}
+              </p>
+            </div>
+
+            {/* first divider */}
+            <div className="w-[320px]  border-[1px] border-[#D6DDEB] mx-[0.7rem]"></div>
+            {priceImpression.map((p) => (
+              <ul
+                key={p.id}
+                className="flex flex-col gap-[22px]  p-[1rem] mx-[0.5rem]"
+              >
+                <li className="gap-[6px] flex items-center ">
+                  <Image
+                    src={p.img}
+                    alt="good"
+                    className="w-[32px] h-[32px] "
+                  />{" "}
+                  <h1 className="font-[600] font-epilogue text-[#1B1B1B] text-[14px] leading-[22px]">
+                    {p.text}
+                  </h1>
+                </li>
+              </ul>
+            ))}
+
+            {/* second divider */}
+            <div className="w-[320px]  border-[1px] border-[#D6DDEB] mx-[0.7rem]"></div>
+
+            <div className="flex flex-col items-center justify-center  gap-[10px] my-[2rem]">
+              <h1 className="font-[700] text-[#002360] md:text-[30px] md:leading-[48px] font-epilogue">
+                #50,000
+              </h1>
+              <button
```

Comment:
there is a button component, please make use of this here

Reply:
i added the button to the make payment, are you saying i should replace the h1 with button?

Reply:
i added the button to the make payment, are you saying i should replace the h1 with button also am lost?